app/representers/api/account_representer.rb

```diff
@@ -18,11 +18,18 @@ class Api::AccountRepresenter < Api::BaseRepresenter
   link :image do
     {
       href: api_account_account_image_path(represented),
-      title: represented.image.try(:filename)
-    } if represented.id
+      title: represented.image.filename
+    } if represented.id && represented.image
   end
   embed :image, class: Image, decorator: Api::ImageRepresenter
 
+  link 'create-image' do
+    {
+      href: api_account_account_image_path(represented),
+      title: 'Create an image'
+    } if represented.id && !represented.image
```

Comment:
What's the case for not always including this link?  Just to cut down on payload?  (I myself may have made that argument).